RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -91,8 +92,15 @@ private extension CustomerCenterView {
         if viewModel.hasSubscriptions {
             if viewModel.subscriptionsAreFromApple,
                let screen = configuration.screens[.management] {
-                ManageSubscriptionsView(screen: screen,
-                                        customerCenterActionHandler: viewModel.customerCenterActionHandler)
+                if ignoreAppUpdateWarning || viewModel.appIsLatestVersion {
+                    ManageSubscriptionsView(screen: screen,
+                                            customerCenterActionHandler: viewModel.customerCenterActionHandler)
+                } else {
+                    AppUpdateWarningView(
+                        onUpdateAppClick: { viewModel.onAppUpdateClick() },
+                        onContinueAnywayClick: { ignoreAppUpdateWarning = true }
+                    )
+                }
```

Comment:
When you click continue the transition is instant which feels a bit abrupt. This can be animated by putting the entire if statement in a ZStack and applying an animation modifier:

```swift
ZStack {
// if { ... } else { ... }
}.animation(.easeInOut, value: ignoreAppUpdateWarning)
```

Reply:
Much better, thanks for the suggestion! https://github.com/RevenueCat/purchases-ios/pull/4193/commits/eac6e3e00c9e465b29f2b4533011f1ba390ef834